pkg/controller/certificates/sync.go

```diff
@@ -351,12 +351,14 @@ func (c *Controller) issue(ctx context.Context, issuer issuer.Interface, crt *v1
 		return nil
 	}
 
-	if _, err := c.updateSecret(crt, crt.Namespace, resp.Certificate, resp.PrivateKey, resp.CA); err != nil {
+	secret, err := c.updateSecret(crt, crt.Namespace, resp.Certificate, resp.PrivateKey, resp.CA)
+	if err != nil {
```

Comment:
nit: this line doesn't need changing

Reply:
Yes it does because secret needs to be passed into the metric function below. When this line uses the `if` block style, `secret` is only available within the scope of that `if` block.

Reply:
d'oh - sorry I didn't notice that!